lib/avromatic/model/types/type_factory.rb

```diff
@@ -26,6 +27,7 @@ module TypeFactory
           'date' => Avromatic::Model::Types::DateType.new,
           'timestamp-micros' => Avromatic::Model::Types::TimestampMicrosType.new,
           'timestamp-millis' => Avromatic::Model::Types::TimestampMillisType.new,
+          'decimal' => Avromatic::Model::Types::DecimalType.new,
```

Comment:
Could we conditionally add this based on `Avro::VERSION` so that we only support decimals in Avro 1.11.0+?